app/consumers/recommendation_consumer.rb

```diff
@@ -0,0 +1,19 @@
+class RecommendationConsumer
+  include Hutch::Consumer
+
+  consume "copro.recommendations.*"
```

Comment:
we can have multiple wild cards. Since it seems like we want model and the action, it could be like `copro.recommendations.*.*` . I feel like this is the more natural way to handle it with routing keys rather than splitting